internal/plans/internal/planproto/planfile.proto

```diff
@@ -116,6 +116,7 @@ enum Action {
     DELETE = 5;
     DELETE_THEN_CREATE = 6;
     CREATE_THEN_DELETE = 7;
+    FORGET = 8;
```

Comment:
I wonder if this breaks compatibility with TACOS and/or makes planfiles unreadable by Terraform?

Reply:
As far as I'm aware, most TACOS read the json output using a library similar to [terraform-json](https://github.com/hashicorp/terraform-json) and work from the json representation of the plan and not this.

Sidenote: once this is in, we should put in a PR to that repo to add the Forget action